poseidon/src/generate_parameters.rs

```diff
@@ -1,3 +1,5 @@
+#![cfg_attr(not(feature = "std"), no_std)]
```

Comment:
there is just no way this will every work with std right? There are io operations - therefore I see no point in pretending it will.

Reply:
Yeah, but we have to mark it for `cargo nono`